### PR TITLE
geoip: Fix compilation on mac os

### PR DESCRIPTION
**Pull request by mebsout**
Add include paths and library paths where lib is installed with either macports or homebrew.
(Not tested with homebrew but should be ok.)

### CODE REVIEW

File: packages/geoip/geoip.0.0.3/opam [hunk 1]
```diff
@@ -5,29 +5,23 @@ homepage: "https://ygrek.org/p/ocaml-geoip/"
 doc: ["https://ygrek.org/p/ocaml-geoip/api/index.html"]
 bug-reports: "https://github.com/ygrek/ocaml-geoip/issues"
 dev-repo: "git://github.com/ygrek/ocaml-geoip.git"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
 tags: ["org:ygrek"]
 build: [
   ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
-  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-build" "-cflags" conf-libgeoip:cflags]
```
Comment (kit-ty-kate) [inline packages/geoip/geoip.0.0.3/opam:12]: are you sure the `-cflags -I<path>` worked in the first place?

Comment (mebsout) [reply packages/geoip/geoip.0.0.3/opam:12]: Indeed, it does not work with this. It needs `-cflags "-I <path>"`, not sure if this is possible with pkg-config. Also I did not find a way to pass -L options to the linker with ocamlbuild, but it works by setting the env variable `LIBRARY_PATH` (with `build-env:`). Again, not sure how to do this with a conf package.

Comment (avsm) [reply packages/geoip/geoip.0.0.3/opam:12]: best off porting geoip to dune and using a configure.ml as other packages do in order to do the directory probing to find out where geoip is installed. This one's not the best thing to fix from the opam packaging unfortunately...

**Comment by mseri** [comment]
I have opened an issue on geoip. I will close this PR

**Pull request closed without merging**